src/gui_gtk.c

```diff
@@ -619,8 +624,10 @@ static void set_labels(GtkLabels *glab, Labels *data)
 	gtk_label_set_text(GTK_LABEL(glab->gtktrad[FRAMGPU1]), data->objects[FRAMGPU1 + opts->selected_gpu]);
 	if(data->gpu_count == 0)
 		gtk_widget_hide(GTK_WIDGET(glab->gridcards));
-	for(i = 0; i < data->gpu_count; i++)
+	for(i = 0; i < data->gpu_count; i++) {
```

Comment:
Please keep the same formatting style, i.e. `{` on a new line (to be aligned with the corresponding `}`).